editor/import/resource_importer_scene.cpp

```diff
@@ -1891,6 +1891,40 @@ void ResourceImporterScene::_replace_owner(Node *p_node, Node *p_scene, Node *p_
 	}
 }
 
+Array ResourceImporterScene::_get_skinned_pose_transforms(ImporterMeshInstance3D *p_src_mesh_node) {
+	Array skin_pose_transform_array;
+
+	// Get an array of bone transforms to be used to rescale the actual mesh.
+	const Ref<Skin> skin = p_src_mesh_node->get_skin();
+	if (skin.is_valid()) {
+		NodePath skeleton_path = p_src_mesh_node->get_skeleton_path();
+		const Node *node = p_src_mesh_node->get_node_or_null(skeleton_path);
+		const Skeleton3D *skeleton = Object::cast_to<Skeleton3D>(node);
+		if (skeleton) {
+			int bone_count = skeleton->get_bone_count();
+			int bind_count = skin->get_bind_count();
+
+			if (bone_count == bind_count) {
```

Comment:
This comparison is not needed, its expected the numbers may differ when coming from Maya.